Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:20-alpine
+
+WORKDIR /usr/app
+
+COPY . .
+
+RUN yarn
```

Comment:
I would propose to use `yarn install --frozen-lockfile` since we have locked all our dependencies in `yarn.lock`